projects/hee-shared/src/lib/analytics/analytics.module.spec.ts

```diff
@@ -2,11 +2,11 @@ import { TestBed, async } from "@angular/core/testing";
 import { AnalyticsModule, AnalyticsConfig } from "./analytics.module";
 import { RouterTestingModule } from "@angular/router/testing";
 import { Router } from "@angular/router";
-import { Location } from "@angular/common";
+// import { Location } from "@angular/common";
```

Comment:
This should be deleted rather than commented if not needed. Same with the other instances.